### ISSUE TITLE
Index table filters trigger the first form in the page when hitting 'return' in the search box

### ISSUE THREAD

**Issue by prioux** [aces/cbrain#1306]
A POST to a controller is performed when it shouldn't happen.

To reproduce:

1) Go the an index page such as `/tasks`
2) Click on the small magnifying glass for one of the columns (e.g. the Task Type column)
3) Enter dummy text in the search box for the column (e.g. 'blah')
4) Hit return

In the case of the `/tasks` page, a request is posted to `TasksController#update_multiple` using the first hijacker submit button in the 'Update Attributes' button/menu.

See screenshots:

![Screen Shot 2023-02-27 at 13 56 12](https://user-images.githubusercontent.com/777588/221658094-32cd0516-be45-4cf9-a7da-8d28dc9c8a15.png)

![Screen Shot 2023-02-27 at 13 57 17](https://user-images.githubusercontent.com/777588/221658123-e2c4ae85-0309-42e3-b772-36b33d7a80bc.png)

**Comment by prioux** [comment]
Solution: completely disable submit on `Enter` in these little search boxes, the return key is not supposed to do anything.

**Comment by prioux** [comment]
The Rails server logfile show the request that was sent:

```
Processing by TasksController#update_multiple as HTML
  Parameters: {"utf8"=>"✓", "authenticity_token"=>"[FILTERED]", "cbrain_task"=>{"group_id"=>"", "results_data_provider_id"=>"27", "tool_config_id"=>""}, "commit"=>"Update", "dup_bourreau_id"=>"21", "archive
_dp_id"=>"", "per_page"=>"25", "update_group_id"=>"1"}
```

**Comment by MontrealSergiy** [comment]
temporary withdrow

**Comment by prioux** [comment]
Fixed by #1307
